netci.groovy

```diff
@@ -214,6 +214,7 @@ def static getJobName(def configuration, def architecture, def os, def scenario,
         suffix += '_bld'
     }
     def baseName = ''
+    def jit = 'ryujit'
```

Comment:
If this is only used in the x86 cases, and it is reset in the lb case, do we really need it to be defined here? Do we use it somewhere else?